bolt/org_config.go

```diff
@@ -0,0 +1,236 @@
+package bolt
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/boltdb/bolt"
+	"github.com/influxdata/chronograf"
+	"github.com/influxdata/chronograf/bolt/internal"
+)
+
+// Ensure OrganizationConfigStore implements chronograf.OrganizationConfigStore.
+var _ chronograf.OrganizationConfigStore = &OrganizationConfigStore{}
+
+// OrganizationConfigBucket is used to store chronograf organization configurations
+var OrganizationConfigBucket = []byte("OrganizationConfigV1")
+
+// OrganizationConfigStore uses bolt to store and retrieve organization configurations
+type OrganizationConfigStore struct {
+	client *Client
+}
+
+func (s *OrganizationConfigStore) Migrate(ctx context.Context) error {
+	return nil
+}
+
+// Get retrieves an OrganizationConfig from the store
+func (s *OrganizationConfigStore) Get(ctx context.Context, orgID string) (*chronograf.OrganizationConfig, error) {
+	var cfg chronograf.OrganizationConfig
+
+	err := s.client.db.View(func(tx *bolt.Tx) error {
+		return s.get(ctx, tx, orgID, &cfg)
+	})
+
+	if err != nil {
+		return nil, err
+	}
+
+	return &cfg, nil
+}
+
+func (s *OrganizationConfigStore) get(ctx context.Context, tx *bolt.Tx, orgID string, cfg *chronograf.OrganizationConfig) error {
```

Comment:
What was the reason for passing in `cfg` as a parameter rather than just returning it?

Something like
```go
func get(ctx context.Context, tx *bolt.Tx, orgID string) (*chronograf.OrganizationConfig, error)
```

so that the function signature matches the `Get` more closely.

Reply:
i wasn't sure which way to go so we wanted to get feedback from you :) this way, it's the controller `Get` function that allocates memory for the cfg, and the helper `get` simply does its one operation and passes that data into what it's given, rather than initializing its own cfg -- thoughts?